sale_cutoff_time_delivery/models/cutoff_time_mixin.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class CutoffTimeMixin(models.AbstractModel):
+
+    _name = "cutoff.time.mixin"
+
+    order_delivery_cutoff_hours = fields.Selection(
+        selection="_selection_order_delivery_cutoff_hours", required=True, default="12"
+    )
+    order_delivery_cutoff_minutes = fields.Selection(
+        selection="_selection_order_delivery_cutoff_minutes",
```

Comment:
Please change to lambda

Reply:
AFAICS, this is not needed, as a string arg is called using using `getattr`: https://github.com/odoo/odoo/blob/13.0/odoo/fields.py#L2170

Reply:
I think this is what you wanted to link: https://github.com/odoo/odoo/blob/45afa3a20693ebed11ad2fa7ebd1b73772ca5b3d/odoo/fields.py#L2242-L2243 ;)